controllers/content.js

```diff
@@ -49,14 +49,27 @@ router.get('/', function (req, res, next) {
             json: true
         }).then(results => {
             // Include _id field in _source
-            const searchResults = results.hits.hits.map(x => {
+            var searchResults = results.hits.hits.map(x => {
```

Comment:
I would prefer `let`instead of `var`. The rest is okay, @kremer-io should also approve it before merging